lib/ansible/modules/network/nxos/nxos_install_os.py

```diff
@@ -55,39 +57,56 @@
         description:
             - Name of the system (or combined) image file on flash.
         required: true
+    timeout:
```

Comment:
Describing `timeout` as an option of this module in the documentation might confuse user since there doc already has `timeout` option from the `provider`/`top` spec . `timeout` description should be in the `notes` section.

Reply:
Yeah, I went back and forth on this.  The only reason I included this here is because it's critical to successful usage of this module.  I have the warning if the setting is not correct though so I will move it to the notes section.

Reply:
Done.